js/bybit.js

```diff
@@ -2925,41 +2951,57 @@ module.exports = class bybit extends Exchange {
             params = this.omit (params, 'position_idx');
         }
         if (market['linear']) {
-            const reduceOnly = this.safeValue2 (params, 'reduce_only', 'reduceOnly', false);
-            const closeOnTrigger = this.safeValue2 (params, 'reduce_only', 'reduceOnly', false);
-            request['reduce_only'] = reduceOnly;
-            request['close_on_trigger'] = closeOnTrigger;
-        }
-        if (price !== undefined) {
-            request['price'] = parseFloat (this.priceToPrecision (symbol, price));
+            request['reduce_only'] = this.safeValue2 (params, 'reduce_only', 'reduceOnly', false);
```

Comment:
this sends `"reduce_only": undefined` to the endpoint if the user doesn't pass it in. I thought that was something undesirable that we didn't want to do

@kroitor your opinion?

Reply:
@samgermain 'false' is passed as the default value, no?